<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>817</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="baseInput"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelOutputBase">
       <property name="font">
        <font>
         <family>Franklin Gothic Medium Cond</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Итоговая СС</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="result">
       <property name="font">
        <font>
         <family>Rockwell Condensed</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Результат</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelValue">
       <property name="font">
        <font>
         <family>Franklin Gothic Medium Cond</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Введите значение</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="exit">
       <property name="font">
        <font>
         <family>Franklin Gothic Demi</family>
        </font>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="getAns">
       <property name="font">
        <font>
         <family>Franklin Gothic Demi</family>
        </font>
       </property>
       <property name="text">
        <string>Получить ответ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="baseOutput"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelInputBase">
       <property name="font">
        <font>
         <family>Franklin Gothic Medium Cond</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Начальная СС</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="valueInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>116</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiondsad">
   <property name="text">
    <string>dsad</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
